presets/4.5/rates/ZoeFPV_3D_rates.txt

```diff
@@ -0,0 +1,94 @@
+#$ DESCRIPTION: -----------
+#$ DESCRIPTION: [YouTube](www.youtube.com/@ZoeFPV)
+#$ DESCRIPTION: [Instagram] (https://www.instagram.com/zoefpv/)
+#$ DESCRIPTION: [Discord with 3D Pilots around to help](https://discord.com/invite/nRn9gU3)
+#$ DESCRIPTION: <br /><br />
+#$ DESCRIPTION:
+#$ DESCRIPTION: ZoeFPV 3D rates
+#$ DESCRIPTION: -----------
+#$ DESCRIPTION: Zoes legendary rate profile that she’s used for years, along with 2 variations of it depending on your own style and feel, plus KittFPV’s rates for a more linear and locked in experience. 
+#$ DESCRIPTION: Each rate has been tested for it’s ability to do chaotic 3D maneuvers, and for tight tic-tocs.
+#$ DESCRIPTION: 
+#$ DESCRIPTION: <br /><br />
+#$ DESCRIPTION:
+#$ WARNING: This rate preset will overwrite all the rate profiles 
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/424
+
+#$ INCLUDE: presets/4.3/rates/defaults.txt
+
+# -- End Defaults --
+# -- Begin ZoeFPV Rates --
+
+rateprofile 0
+
+# rateprofile 0
+set rateprofile_name = Z' Magic
+set thr_expo = 25
+set roll_rc_rate = 4
+set pitch_rc_rate = 4
+set yaw_rc_rate = 4
+set roll_expo = 33
+set pitch_expo = 33
+set yaw_expo = 33
+set roll_srate = 80
+set pitch_srate = 80
+set yaw_srate = 80
+
+rateprofile 1
+
+# rateprofile 1
+set rateprofile_name = Z' Buttr
+set thr_expo = 33
+set roll_rc_rate = 6
+set pitch_rc_rate = 6
+set yaw_rc_rate = 6
+set roll_expo = 42
+set pitch_expo = 42
+set yaw_expo = 42
+set roll_srate = 74
+set pitch_srate = 74
+set yaw_srate = 74
+
+rateprofile 2
+
+# rateprofile 2
+set rateprofile_name = Z's OG
+set roll_rc_rate = 17
+set pitch_rc_rate = 17
+set yaw_rc_rate = 17
+set roll_expo = 58
+set pitch_expo = 58
+set yaw_expo = 58
+set roll_srate = 74
+set pitch_srate = 74
+set yaw_srate = 74
+
+rateprofile 3
+
+# rateprofile 3
+set rateprofile_name = KittFPV
+set thr_expo = 33
+set roll_rc_rate = 20
+set pitch_rc_rate = 20
+set yaw_rc_rate = 17
+set roll_expo = 42
+set pitch_expo = 42
+set yaw_expo = 31
+set roll_srate = 62
+set pitch_srate = 62
+set yaw_srate = 43
+
+# restore original rateprofile selection
+rateprofile 2

```

Comment:
why rateprofile 2 by default?

Reply:
because that's her OG rates as it says in the rate name